doc/contributing/maintaining-openssl.md

```diff
@@ -7,15 +7,12 @@ currently need to generate four PRs as follows:
 
 * a PR for master which is generated following the instructions
   below for OpenSSL 3.x.x.
+* a PR for 18.x following the instructions in the v18.x-staging version
+  of this guide.
 * a PR for 16.x following the instructions in the v16.x-staging version
   of this guide.
 * a PR for 14.x following the instructions in the v14.x-staging version
   of this guide.
-* a PR which uses the same commit from the third PR to apply the
```

Comment:
Not entirely sure about this remotion. Is it still valid?

Reply:
It's valid until support for Node.js 12 ends at the end of this month.